tools/CMakeLists.txt

```diff
@@ -40,13 +40,21 @@ configure_file(${CMAKE_CURRENT_SOURCE_DIR}/oio-sds.pc.in
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/oio-bootstrap.py
 	${CMAKE_CURRENT_BINARY_DIR}/oio-bootstrap.py @ONLY)
 
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/oio-sds.pc
+install(FILES
+			${CMAKE_CURRENT_BINARY_DIR}/oio-sds.pc
 		DESTINATION ${PKGCONFIG_DIRECTORY}
 		PERMISSIONS OWNER_WRITE OWNER_READ GROUP_READ WORLD_READ)
 
-install(TARGETS oio-tool DESTINATION bin)
+install(TARGETS
+			oio-tool
+		DESTINATION bin)
 
-install(TARGETS oio-file DESTINATION bin)
+install(TARGETS
+			oio-file
+			oio-lb-benchmark
```

Comment:
This tool is still a bit rude, but it can help admins test their service pool configuration. I vote for it to be installed on production environments.